web_src/js/features/repo-branch.js

```diff
@@ -13,6 +21,13 @@ export function initRepoBranchButton() {
     }
 
     $(fromSpanName).text($(this).attr('data-branch-from'));
-    $($(this).attr('data-modal')).modal('show');
+  });
+}
+
+function initRepoRenameBranchButton() {
+  $('.show-rename-branch-modal').on('click', function () {
+    const oldBranchName = $(this).attr('data-old-branch-name');
+    $(renameBranchFromInputSelector)?.val(oldBranchName);
```

Comment:
jQuery doesn't work this way (`$(...)?.xxx()`)

Reply:
Yes, this `?` has no effect because empty jQuery selection will still be truthy.